packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,19 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private updateTitle(){
+        // Update the page <title> when the hash changes
+        let title:string;
+        if(this.state.activePageId === 'blueprint' && this.state.activeSectionId === 'blueprint'){
+            const indexTitle: string = 'Blueprint - Documentation';
+            title = indexTitle;
```

Comment:
Another option is just to set `document.title = indexTitle` right here. The function is simple enough that spelling out the assignment twice isn't overkill.